### PR TITLE
Fix PNG aliasing

### LINKED ISSUE #758: Portable Network Graphics (PNG) still refers to old version (2003)
A [specref search for PNG](https://www.specref.org/?q=PNG) resolves **[PNG]** to the old, 2003 Second Edition. 

There is also an explicit [PNG2e] which is an alias to [PNG], and assorted dated references to [PNG third edition](https://www.w3.org/TR/png-3/) which is currently a Candidate Recommendation and has [a bunch of new features, clarifications and corrections](https://www.w3.org/TR/png-3/#F-ChangeList) compared to the second edition.

Instead, [PNG 2e] should alias to the dated second edition, and there should be a [PNG 3e] which aliases to the latest.

On the W3C TR page, going to /TR/PNG takes you to the third edition.

I know that the data from /TR is [auto updated](https://github.com/tobie/specref#hourly-auto-updating) so I assume something on the W3C side is not correctly indicating that the third edition supersedes the second edition. I would like to get this cleared up and assume that manual editing is not the correct way forward here as it will get overwritten?

### PR DESCRIPTION
**Pull request by tobie**
Closes #758

/cc @svgeesus 

I think this is what you're asking for. Leverages an undocumented (sorry) feature to force overwrites for exactly those usecases.

### CODE REVIEW

**Comment by tobie** [comment]
Something's off unfortunately. How urgent is this?

**Comment by svgeesus** [comment]
No rush, I just happened to notice it in a spec today, so raised the issue before I forgot.

File: refs/w3c.json [hunk 1]
```diff
@@ -7414,6 +7417,12 @@
         "edDraft": "https://w3c.github.io/PNG-spec/",
         "repository": "https://github.com/w3c/PNG-spec"
     },
+    "PNG-19961001": {
+        "aliasOf": "png-1-19961001"
+    },
+    "PNG-20031110": {
+        "aliasOf": "png-1-20031110"
+    },
```
Comment (deniak) [inline refs/w3c.json:7425]: I think the main issue is a mix of uppercase/lowercase shortnames and a missing version"

```suggestion
    "PNG-19961001": {
        "aliasOf": "PNG-1-19961001"
    },
    "PNG-20030520": {
        "aliasOf": "PNG-1-20030520"
    },
    "PNG-20031110": {
        "aliasOf": "PNG-1-20031110"
    },
```

File: refs/w3c.json [hunk 2]
```diff
@@ -98431,6 +98440,45 @@
...
+                "title": "Portable Network Graphics (PNG) Specification (Second Edition)",
+                "rawDate": "2003-11-10",
+                "status": "REC",
+                "publisher": "W3C",
+                "hasErrata": "https://www.w3.org/2003/11/REC-PNG-20031110-errata",
+                "source": "https://www.w3.org/2002/01/tr-automation/tr.rdf"
+            }
+        },
+        "repository": "https://github.com/w3c/PNG-spec"
+    },
```
Comment (deniak) [inline refs/w3c.json:98481]: Not needed since we already have `PNG-1`

```suggestion
```

**Review by deniak — CHANGES_REQUESTED** [review]
The PNG2e alias should also be updated in `refs/biblio.json`:
```
     "PNG2e": {
         "aliasOf": "PNG-1"
     },
```

**Comment by deniak** [comment]
@tobie, I made some suggestions which I believe should work.
I see specref still relies on tr.rdf which we plan to retire in the future. Indeed, tr.rdf has some limitations and doesn't show data related to the specification series. Instead, the [W3C API](https://w3c.github.io/w3c-api/) provides more details, e.g. https://api.w3.org/specification-series/png.

That said, I'm aware upgrading specref to use the API isn't a priority but I'm just mentioning in case we can find someone to work on that.

**Comment by tobie** [comment]
Thanks @deniak. Your changes would be overwritten on next update, unfortunately.

**Comment by deniak** [comment]
> Thanks @deniak. Your changes would be overwritten on next update, unfortunately.

Right, I forgot the references get automatically updated.
Changing the `newId` for `PNG` in `overwrites/w3c.json` should do the trick I guess:
```
  { "id": "PNG",               "action": "renameTo", "newId": "PNG-1"},
  { "id": "PNG",               "action": "createAlias", "aliasOf": "png-3"}
```

**Comment by tobie** [comment]
Have you tried it? I wasn’t working for me yesterday.

**Comment by deniak** [comment]
> Have you tried it? It wasn’t working for me yesterday.

I just gave it a try and it seems to work.
I don't have write access to the repo, but I'm happy to open a new PR if you want.

**Comment by tobie** [comment]
Adding your changes here would be easier.

**Comment by deniak** [comment]
> Adding your changes here would be easier.

Done.

**Comment by tobie** [comment]
Sorry, I hadn't thought this through much and didn't consider you'd have to force push to clean things up. Apologies!

**Comment by deniak** [comment]
no worries :)

**Comment by deniak** [comment]
Thanks @tobie

**Comment by tobie** [comment]
No, thank you!

**Pull request merged**